src/GraphQL.SystemReactive/SubscriptionExecutionStrategy.cs

```diff
@@ -1,178 +0,0 @@
-using System.Reactive.Linq;
-using GraphQL.Subscription;
-using GraphQL.Types;
-using GraphQLParser.AST;
-
-namespace GraphQL.Execution
-{
-    /// <summary>
-    /// Executes a subscription.
-    /// </summary>
-    public class SubscriptionExecutionStrategy : ParallelExecutionStrategy
-    {
-        /// <summary>
-        /// Gets a static instance of <see cref="SubscriptionExecutionStrategy"/> strategy.
-        /// </summary>
-        public static new SubscriptionExecutionStrategy Instance { get; } = new SubscriptionExecutionStrategy();
-
-        /// <summary>
-        /// Executes a GraphQL subscription request and returns the result. The result consists
-        /// of one or more streams of GraphQL responses, returned within <see cref="SubscriptionExecutionResult.Streams"/>.
-        /// No serializable <see cref="ExecutionResult"/> is directly returned.
-        /// </summary>
-        public override async Task<ExecutionResult> ExecuteAsync(ExecutionContext context)
-        {
-            var rootType = GetOperationRootType(context);
-            var rootNode = BuildExecutionRootNode(context, rootType);
-
-            var streams = await ExecuteSubscriptionNodesAsync(context, rootNode.SubFields!).ConfigureAwait(false);
-
-            ExecutionResult result = new SubscriptionExecutionResult
-            {
-                Executed = true,
-                Streams = streams
-            }.With(context);
-
-            return result;
-        }
-
-        private async Task<IDictionary<string, IObservable<ExecutionResult>>> ExecuteSubscriptionNodesAsync(ExecutionContext context, ExecutionNode[] nodes)
-        {
-            var streams = new Dictionary<string, IObservable<ExecutionResult>>();
-
-            foreach (var node in nodes)
-            {
-                var stream = await ResolveEventStreamAsync(context, node).ConfigureAwait(false);
-
-                if (stream != null)
-                    streams[node.Name!] = stream;
-            }
-
-            return streams;
-        }
-
-        /// <summary>
-        /// Asynchronously returns a stream of <see cref="ExecutionResult"/> responses for the
-        /// specified <see cref="ExecutionNode"/>.
-        /// </summary>
-        protected virtual async Task<IObservable<ExecutionResult>?> ResolveEventStreamAsync(ExecutionContext context, ExecutionNode node)
-        {
-            context.CancellationToken.ThrowIfCancellationRequested();
-
-            object? source = (node.Parent != null)
```

Comment:
I cannot find usages of `source` here.

Reply:
You are correct; `source` was not used previously.

Reply:
wow